umbral-pre-wasm/src/lib.rs

```diff
@@ -66,12 +66,16 @@ impl CapsuleFrag {
         signing_pubkey: &PublicKey,
         delegating_pubkey: &PublicKey,
         receiving_pubkey: &PublicKey,
+        metadata: Option<Box<[u8]>>,
     ) -> bool {
+        // feels like there should be a better way...
```

Comment:
Maybe we could pass `metadata: Option<&[u8]>` in here?

Reply:
You mean to this method? I wish I could, but `wasm-pack` doesn't seem to support it. My comment was mostly referring to the horrible conversion below.